DigitalLearningSolutions.Data/DataServices/SupervisorDelegateDataService.cs

```diff
@@ -79,7 +80,7 @@ IEnumerable<string> emails
                     INNER JOIN AdminUsers au ON sd.SupervisorAdminID = au.AdminID
                     WHERE au.CentreID = @centreId
                       AND sd.DelegateEmail IN @emails
-                      AND sd.DelegateUserID IS NULL
+                      AND sd.CandidateID_deprecated IS NULL

```

Comment:
This change needs reverting. It doesn't appear to relate to this ticket and is replacing new code with old.